src/main.ts

```diff
@@ -73,7 +73,7 @@ async function start () {
 // execution and error handling
 start().catch(error => {
   console.error(error) // eslint-disable-line no-console
-  alert('There was an error starting this page. (See console for details.)\n' +
+  window.alert('There was an error starting this page. (See console for details.)\n' +
```

Comment:
Although the previous code was equivalent, I encountered a "window.alert is not defined" error and couldn't find any references to window.alert (because it was simply called "alert"). But now we will!